### DISCUSSION TITLE
Showcase of the usage of the library

### DISCUSSION THREAD

**Discussion by jcubic** [peggyjs/peggy#231] [Ideas]
Just an idea, it would nice if there is showcase of project that use the library. I could be used as reference. For most stared project [jQuery Terminal](https://terminal.jcubic.pl/examples.php#wild) I have a long list of projects where users can see live usage.

I myself would gladly be on the list. My project [Gaiman](https://gaiman.js.org/) have a nice live tool that I've created that help me working on the parser.

It generate the parser on the fly, that generate JS AST, that then is converted to the JavaScript code that executed inside an iframe with help of service worker. All those things are interested while in Dev Mode. The project is on the GitHub pages so everything happen in browser.

**Comment by jcubic** [comment]
Oh, I've found this on a Wiki. [Projects Using Peggy](https://github.com/peggyjs/peggy/wiki/Projects-Using-Peggy). Can you add Gaiman to the list?

**Reply by hildjj** [reply]
The wiki is now editable by all!  (Bug I added Gaiman for you)

**Comment by jcubic** [comment]
I know that it's editable, not sure how. I've added my [tutorial](https://github.com/peggyjs/peggy/wiki/Tutorials) 17 days ago. It showcases PEG.js, because this is what I was using when I've written this article but there is a Note at the end about Peggy.

**Reply by hildjj** [reply]
Nod, sorry, I was just going through old discussions and replying.  Thanks for your edit!
